org.nextcloud.Nextcloud.json

```diff
@@ -13,6 +13,7 @@
         "--share=network",
         "--share=ipc",
         "--filesystem=home:rw",
+        "--filesystem=xdg-cache",
```

Comment:
Why are you passing this through?

Reply:
My thought was to share the lock with other running nextcloud processes. Is that also the case if we don't pass the users xdg-cache through?

Reply:
It is not needed, the lock file will be in a readable location eitherway, the only thing this does is expose other applications caches.

Reply:
OK, thanks for having a look. Fixed that.